.eleventy.js

```diff
@@ -79,7 +79,19 @@ module.exports = (eleventyConfig) => {
     collection
       .getAll()
       .filter((item) => item.data.client)
-      .sort((a, b) => b.date - a.date),
+      .sort((a, b) => {
+        if (!a.data.end) {
+          if (a.date > b.data.end) {
+            return -1;
+          }
+        }
+        if (!b.data.end) {
+          if (a.data.end > b.date) {
+            return -1;
+          }
+        }
+        return b.data.end - a.data.end;
+      }),
```

Comment:
@dvdherron This is close, but I think probably we want ongoing clients (without an "end" date) to appear first in the list. That means we'll need to add end dates to old clients, but I can do that. I'll try something like:

```js
        if (!a.data.end) {
          return b.data.end ? -1 : b.date - a.date;
        }
        if (!b.data.end) {
          return 1;
        }
```